### PR TITLE
[FLN-2018] Adding Florianópolis in DevOpsDays events.

### PR DESCRIPTION
**Pull request by perylemke**
Hi Everyone,

Follow the PR with Florianópolis in the cities to take a DevOpsDays.

Best regards,

### CODE REVIEW

File: data/events/2018-florianopolis.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    email: "sergio.mendonca@gmail.com"
+    employer: "Mettzer"
+  - name: "Jonatas Baldin"
+    email: "jonatas.baldin@gmail.com"
+    twitter: "jonatasbaldin"
+    employer: "Maecenas"
+  - name: "Luciano Faustino"
+    email: "lucianoborguetti@gmail.com"
+    employer: "Neoway"
+organizer_email: "organizers-floripa@devopsdays.org" # Put your organizer email address here
```
Comment (gomex) [inline data/events/2018-florianopolis.yml]: please change to organizers-floripa-2018

File: data/events/2018-florianopolis.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+proposal_email: "proposals-floripa@devopsdays.org" # Put your proposal email address here
```
Comment (gomex) [inline data/events/2018-florianopolis.yml]: please change to proposals-floripa-2018@devopsdays.org

File: data/events/2018-florianopolis.yml (hunk 1 above)
Comment (perylemke) [reply data/events/2018-florianopolis.yml]: Fixed :)

File: data/events/2018-florianopolis.yml (hunk 2 above)
Comment (perylemke) [reply data/events/2018-florianopolis.yml]: Fixed! :)

**Comment by bridgetkromhout** [comment]
The deploy is still processing and I need to head out, but that may fix the problem and if it does, feel free to merge.

**Pull request merged**